examples/user_guide/03-Customizing_Plots.ipynb

```diff
@@ -339,7 +339,7 @@
     "\n",
     "#### ``%%opts``\n",
     "\n",
-    "As shown in the examples above, customizes a particular HoloViws output displayed in a code cell. This application is not global and persists for that object which means settings stay in place if the object is re-displayed. Only accepts the string specification format. All cell magics need to appear *above* any code in the cells they are used in and cannot be used if there is no code in the cell. Only accepts the string specification syntax.\n",
+    "As shown in the examples above, customizes a particular HoloViews output displayed in a code cell. This application is not global and persists for that object which means settings stay in place if the object is re-displayed. Only accepts the string specification format. All cell magics need to appear *above* any code in the cells they are used in and cannot be used if there is no code in the cell. Only accepts the string specification syntax. Only applied to the output of the cell when it is displayed by IPython, and thus will not apply to objects exported or displayed using a separate mechanism before the cell completes, and does not affect objects created but not displayed in the output of the cell.\n",
```

Comment:
Oops; I thought this was just a typo fix, but it's actually a clarification that I meant to have as a separate PR.  @jlstevens should verify the text here.

Reply:
Looks correct to me.